kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/canvas/controls/containment/CaseManagementContainmentAcceptorControlImpl.java

```diff
@@ -241,8 +241,10 @@ int getAddIndex(final WiresShape wiresShape,
 
             // Otherwise, add to a stage vertically
             // And if it is the same stage
-            if (state.getOriginalParent().isPresent() && state.getOriginalParent().get().equals(container)) {
-                return parent.getOutEdges().size() - 1;
+            if (state.getOriginalParent().isPresent()) {
```

Comment:
just  a suggestion on this block:

```suggestion
           return state.getOriginalParent()
                    .filter(container::equals)
                    .map(p -> parent.getOutEdges().size() - 1)
                    .orElse(parent.getOutEdges().size());
```

Reply:
Made a similar suggestion regarding to the same code. Yours is better. :-)

Reply:
Hey @triceo , yeah we made the suggestions almost at the same time, just saw them now :)